Backend/Backend/Program.cs

```diff
@@ -33,9 +33,10 @@ public static void Main(string[] args)
         builder.Services.AddScoped<IProfileService, ProfileService>();
         builder.Services.AddScoped<ISubscriptionService, SubscriptionService>();
         builder.Services.AddScoped<INotificationService, NotificationService>();
-        builder.Services.AddScoped<ISocialService, SocialService>();
```

Comment:
Don't remove this, otherwise the interface wont work.

Reply:
I added it back, nice catch!